config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -227,6 +253,50 @@ periodics:
     testgrid-tab-name: gce-windows-containerd-master
     description: Runs tests on a Kubernetes cluster with Windows containerd nodes on GCE
 
+- name: ci-kubernetes-e2e-windows-node-throughput
+  interval: 2h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-load-gce-windows: "true"
+  annotations:
+    testgrid-dashboards: google-windows
+    testgrid-tab-name: windows-gce-node-throughput
+  decorate: true
+  decoration_config:
+    timeout: 6h
```

Comment:
the default is 2h, if your test takes less than 2h, you don't really need to set it at all.

Reply:
removed